### PR TITLE
Expands Salvage Gantry Species

### PR DESCRIPTION
**Pull request by BladeburstNINJA**
Adds Skrell, Unathi, Yeosa, Shells, Tajarans, and Genemodders to the Salvage Gantry.
:cl:
maptweak: Expands the Salvage Gantry Species to include Skrell, Unathi, Yeosa, Shells, Tajarans, and Genemodders.
/:cl:

### CODE REVIEW

**Review by Superhats — APPROVED** [review]
Code's fine. Still needs a changelog.

**Comment by BurpleBineapple** [comment]
Same issue as before, I still feel that making the Gantry yet another Bearcat instead of a unique human offship isn't the right move. Skrell/Vox have their own species specific offships and I think it would be nice to have one for humans. Shells/Genemodders I suppose are fine since the former is an IPC and the latter is technically a human subtype.

Minor others issues with this is the odd fact of species like Skrell being on board. This is very much a poor scav vessel with no diplomatic power. It seems strange that a Skrell would work here rather than back in its own space.

Tldr; I would prefer if you want to add species to gantry, add Shells and Genemodders but not the other alien species for the sake of by having 1 human only offship

**Review by BurpleBineapple — CHANGES_REQUESTED** [review]
Going to finalize this with saying that, if you do want to expand the species to Gantry add only Genemodders and Shells to keep this a Human/IPC only ship.

**Comment by BurpleBineapple** [comment]
Stale
